covid19-dashboard/main.py

```diff
@@ -12,201 +12,391 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import json
 from collections import defaultdict
-from flask import Flask, send_from_directory
-import os
+from datetime import datetime, timedelta
+from typing import List, Dict, Tuple
+
+from flask import Flask, send_from_directory, Response
 from flask_caching import Cache
-from flask_cors import CORS
+from flask_compress import Compress
+
 from send_request import send_request
+from calculate_data import calculate_data
+
+config = dict(DEBUG=True, CACHE_TYPE="filesystem", CACHE_DIR="/tmp")
 
-config = dict(DEBUG=True,
-              CACHE_TYPE="filesystem",
-              CACHE_DIR='/tmp',
-              CACHE_DEFAULT_TIMEOUT=5000)
+app = Flask(__name__, static_folder="./build")
+
+Compress(app)
 
-app = Flask(__name__, static_folder='./dc-covid19/build')
-CORS(app)
 app.config.from_mapping(config)
 
-# Load the configuration file from the instance folder
-app.config.from_pyfile('config.py')
+# Load the configuration file from the instance folder.
+app.config.from_pyfile("config.py")
 cache = Cache(app)
 
-# Retrieve the API KEY from the configuration file
-API_KEY = app.config['API_KEY']
+# Retrieve the API KEY from the configuration file.
+API_KEY = app.config["API_KEY"]
 
-# Retrives DataCommons Server from configuration file
+# Retrieves DataCommons Server from configuration file.
 DC_SERVER = app.config["DC_SERVER"]
 
-@app.after_request
-def add_header(response):
-    """Adds the Cache Control header to ensure browser stores a cache"""
-    # Keep cache in browser for 4 hours.
-    response.cache_control.max_age = 14400
+# DataCommons Statvars to be used.
+INCREMENTAL_CASES_STATVAR = (
+    "IncrementalCount_MedicalConditionIncident_COVID_19_ConfirmedOrProbableCase"
+)
+INCREMENTAL_DEATHS_STATVAR = (
+    "IncrementalCount_MedicalConditionIncident_COVID_19_PatientDeceased")
+
+# The type of sub-regions. Used to remove trailing from name.
+# For example, "Miami-Dade County" becomes "Miami-Dade".
+PLACE_TYPES = ["County", "Parish", "Borough"]
```

Comment:
Hm, I'm not sure if this is necessary to remove. Was this a requested feature?

Reply:
I added it because the names were getting too long.

Northumberland County, Pennsylvania

But i will remove and will look for a way to request an alternate name from DC KG.